### DISCUSSION TITLE
[General] <Title>

### DISCUSSION THREAD

**Discussion by Robenenym** [zkSync-Community-Hub/zksync-developers#254] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

i cant join zksync discord no matter what, please add me benenymek, thanks

### What is the specific issue or error you're encountering?

dc

### Can you share the error messages or logs you're receiving, if any?

no

### Have you made any recent changes to the contract before encountering this issue?

no

### Are there any external libraries or contracts that your contract interacts with?

no

### Can you provide the relevant portions of your contract code where the issue is occurring?

no

### Have you tried to isolate the problem, and if so, what were the results?

no

### What steps have you already taken to try to resolve the issue?

no

**Comment by 0xsisyfos** [comment]
Hello,

Thank you for reaching out to us. 
- Make sure you have a profile picture in Discord
- Make sure you do not have words such as "admin", "airdrop", etc. in your profile name.
- If you were banned, please fill out the form and we'll review as soon as we can https://appeal.zksync.dev/.

Best regards,
